release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3332,6 +3332,63 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* Previously, when you attempted to install a cluster on {rh-openstack-first} and you used special characters, such as the hash sign (`#`) in a cluster name, the Neutron API failed to tag a security group with the name of the cluster. This caused installation of the cluster to fail. With this release, the installation program uses an alternative endpoint to tag security groups and this endpoint supports the use of special characters in tag names. (link:https://issues.redhat.com/browse/OCPBUGS-37492[*OCPBUGS-37492*])  
+
+* Previously, the Dell iDRAC baseboard management controller (BMC) with the Redfish protocol caused clusters to fail on the Dell iDRAC servers. With this release, an update to the `idrac-redfish` management interface to unset the `ipxe` parameter fixed this issue. (link:https://issues.redhat.com/browse/OCPBUGS-37262[*OCPBUGS-37262*])
+
+* Previously, the `assisted-installer` did not reload new data from the `assisted-service` when the `assisted-installer` checked control plane nodes for readiness and a conflict existed with a write operation from the `assisted-installer-controller`. This conflict caused the `assisted-installer` not detect a node that was marked by the `assisted-installer` as `Ready` because the `assisted-installer` relied on older information. With this release, the `assisted-installer` can receive the newest information from the `assisted-service`, so that it the `assisted-installer` can accurately detect the status of each node. (link:https://issues.redhat.com/browse/OCPBUGS-37167[*OCPBUGS-37167*])
+
+* Previously, the DNS-based egress firewall incorrectly caused memory increases for nodes running in a cluster because of multiple retry operations. With this release, the retry logic is fixed so that DNS pods no longer leak excess memory to nodes. (link:https://issues.redhat.com/browse/OCPBUGS-37078[*OCPBUGS-37078*])
+
+////
+* Need to confirm with MCO team is OCPBUGS-37485 is a known issue or bug fix. 
+* OCPBUGS-37065 is internal and won't be documented. Confirmed with SME to drop the Jira
+* OCPBUGS-36937 is internal and won't be documented. Reporter is on PTO but Confirmed with assignee SME to drop it.
+* OCPBUGS-36227 is internal and won't be documented. Confirmed with SME to drop the Jira
+* OCPBUGS-37621 is not something a customer needs to know. Confirmed with SME to drop the Jira.
+* OCPBUGS-34790 is not something a customer needs to know. Confirmed with SME to drop the Jira.
+////
+
+* Previously, `HostedClusterConfigOeprator` does not delete the `ImageDigestMirrorSet` (IDMS) object after a user removed the `ImageContentSources` field from `HostedCluster` object. This caused the IDMS object to remain in the `HostedCluster` object. With this release, `HostedClusterConfigOeprator` removes all IDMS resources in the `HostedCluster` object so that this issue no longer exists. (link:https://issues.redhat.com/browse/OCPBUGS-36766[*OCPBUGS-36766*])
```

Comment:
```suggestion
* Previously, `HostedClusterConfigOperator` did not delete the `ImageDigestMirrorSet` (IDMS) object after a user removed the `ImageContentSources` field from the `HostedCluster` object. This caused the IDMS object to remain in the `HostedCluster` object. With this release, `HostedClusterConfigOperator` removes all IDMS resources in the `HostedCluster` object so that this issue no longer exists. (link:https://issues.redhat.com/browse/OCPBUGS-36766[*OCPBUGS-36766*])
```
Also, do you need to specify the `HostedClusterConfigOperator` object reference type? Is it a resource, object, spec, etc.?

Reply:
I had a bit of confusing issues with the SME on this but we'll go with "resource".